portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,31 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import java.util.ArrayList;
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/* Servlet that returns some example content.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Josef!");
+    ArrayList<String> shows = new ArrayList<String>();
```

Comment:
nit: Generally a good idea to use the interface as the type (List) as common utility methods with Collections don't return Arraylist. Also, the second 'String' doesn't need to be added as it's inferred, so could just be '= new ArrayList<>();'.
You could also probably also instantiate all of its values in 1 line without losing clarity with:
`List<String> list = Arrays.asList("a", "b", "c");`

Reply:
Another note, if this list will never change, it's an even better idea to use an ImmutableList to make sure of that.

Reply:
Fixed and noted.  Later on the list will change.